cki_lib/lib.sh

```diff
@@ -35,3 +35,20 @@ function cki_yum_tool()
   fi
 }
 
+function print_info()
+{
+  # Print an informational message with a friendly emoji.
+  echo "ℹ️ ${1}"
+}
+
+function print_success()
+{
+  # Print a success message with a friendly emoji.
+  echo "✅ ${1}"
+}
+
+function print_warning()
+{
+  # Print an warning message with a friendly emoji.
+  echo "🟡 ${1}"
```

Comment:
Hm, this emoji doesn't seem to be recognised by either GitHub, or my terminal :cry: I wonder if we should use a different one, for better compatibility. Weirdly, looking further down in the PR, it seems that the emoji used by hand (⚠️) is recognised? So maybe use that one? idk

Also, maybe drop the comments? I don't think they give that much value :sweat_smile:

Reply:
I changed to the yellow circle since the warning triangle wasn't showing up well in some browsers. :/ I'll switch it back and try again.